examples/demo.css

```diff
@@ -264,4 +264,13 @@ a.name-header{
 }
 .col-md-4{
   padding-left: 30px;
+}
+.tools{
+  margin-top: -30px;
```

Comment:
Yes, here i worry a bit, that this is not as good as using `prepend` - because it's not really inside the pane, you know? Can you try using `prepend` to see? thank you!!!

Reply:
Yes using fixed units like pixels in not good.